qa/tasks/cephfs/mount.py

```diff
@@ -166,7 +165,8 @@ def assert_and_log_minimum_mount_details(self):
                               sudo=True).decode())
 
     def is_mounted(self):
-        return self.mounted
+        return self.client_remote.run(args=f'findmnt {self.hostfs_mntpt}',
+                                      check_status=False).returncode == 0
```

Comment:
What diagnostic information is printed if it is/is not mounted? (Is `stderr` captured?)

Reply:
It doesn't print any. I think it is better to leave it to calling function since it would have better context and therefore it would be able to print more accurate, more meaningful message.

Reply:
What will happen when the `mntpt` was mounted by other users ? Should we worry about this ?

Reply:
What will happen if `mntpt` is mounted by different filesystems or filesystem other than ceph?

Reply:
I don't think so because `hostfs_mntpt` is a variable from qa.tasks.cephfs.mount.FuseMount. Therefore, path in it is has to be mounted by vstart_runner.py.